src/api/status/public/js/feed/index.js

```diff
@@ -1,14 +1,14 @@
-const apiHost = new URL(window.location).origin;
+// const apiHost = new URL(window.location).origin;
```

Comment:
Let's not do this.  Either include this code ore remove it, but not comment it out.

Reply:
Removed all unused files